service/src/main/resources/db/changesets/20221108_limits.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: limits
```

Comment:
can you name this sets_max_profile_size and also rename the file so someone doesn't need to click in to know what's going on in here?

Reply:
Sure!